### ISSUE TITLE
More types in the generated docs than in the written docs	

### ISSUE THREAD

**Issue by arboleya** [FuelLabs/fuels-ts#1551]
There were more types mentioned in the API docs than in the written ones, I'd recommend ensuring if we have a new SDK type it gets properly documented in written and generated form.

> [!NOTE]
> - Incoming from https://github.com/FuelLabs/fuels-ts/discussions/1545

**Comment by arboleya** [comment]
Blocked by:
 - #2481

**Comment by danielbate** [comment]
I don't agree with this and would be inclined to close this. The API reference is intended to be a complete reference of the SDK, the docs are a more descriptive guide that takes the user on an intentional journey. Adding every type to the guide would add noise, we wouldn't need the API reference if this was the case. 
> cc @FuelLabs/sdk-ts

**Comment by petertonysmith94** [comment]
@danielbate I agree. Having the API docs external to the docs makes a massive difference and provides a better separation of responsibilities.

**Comment by maschad** [comment]
Agreed as well @danielbate
